build-system/server/new-server/transforms/transform.ts

```diff
@@ -19,15 +19,32 @@ import minimist from 'minimist';
 import posthtml from 'posthtml';
 import transformModules from './modules/modules-transform';
 import transformScriptPaths from './scripts/scripts-transform';
-import transformStories from './stories/stories-transform';
 import transformCss from './css/css-transform';
 
 const argv = minimist(process.argv.slice(2));
-const transforms = [transformStories(), transformScriptPaths(), transformCss()];
+const FOR_TESTING = argv._.includes('integration');
+// Use 9876 if running integration tests as this is the KARMA_SERVER_PORT
+const PORT = FOR_TESTING ? 9876 : 8000;
```

Comment:
Would it make sense to import the port value from either `build-system/server/amp4test.js` or `build-system/tasks/karma.conf.js`, since they're both related to the server?

Reply:
yeah it would make sense, but i know importing from karma.conf.js would have a lot of side effects, so ideally i would want to move it to a purely no side effect module. what would be the best file to move it into?

Reply:
i plan to move it into `build-system/test-configs/config.js`, if you don't mind ill do it in a follow up in https://github.com/ampproject/amphtml/issues/30529 as importing this into a typescript file is a little bit tricky